### DISCUSSION TITLE
Phoebe running only on one node

### DISCUSSION THREAD

**Discussion by mami-deka** [phoebe-project/phoebe2#682] [General]
I am trying to run Phoebe  with PBS.
I would like to run Phoebr  under MPI" for emcee sampling.
I turned on MPI in Phoebe and run in PBS, two nodes and 32 CPUs in per node and then I run the python code
using:
python3 name_of_the_python_code

But the job is running on the first node of the allotted nodes while the second node sitting idle.

Please help me in resolving the issue.

**Comment by kecnry** [comment]
If you write a test program that does not use phoebe (just prints to a file or something), with the same PBS options, does it run as expected across all the assigned nodes?

**Reply by mami-deka** [reply]
Thank you. Yes, the issue is not with phoebe. The test code is also running in one node only.

**Reply by kecnry** [reply]
Ok, let us know if you're able to get that running with your server PBS setup but have any issues when applying it to phoebe.

**Reply by mami-deka** [reply]
Now, Phoebe is working fine in PBS server, but only in one node and multiprocessing mode.
